Sources/Fullscreen/FavoriteAdCommentSheet/FavoriteAdCommentViewController.swift

```diff
@@ -125,6 +124,7 @@ final class FavoriteAdCommentViewController: UIViewController {
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         _ = textView.resignFirstResponder()
+        adView.resetContent()
```

Comment:
Why do we need this?

Reply:
Just to make sure `remoteImageView.cancelLoading()` is called from `FavoriteAdView`